src/main/scala/uk/gov/hmrc/ct/computations/calculations/LowEmissionCarsCalculator.scala

```diff
@@ -20,11 +20,13 @@ trait LowEmissionCarsCalculator extends CtTypeConverters {
 
   def taxPoolForCar(car: Car): String = {
     car.dateOfPurchase match {
-      case Some(d) if d < new LocalDate("2009-04-01") => range1(car)
-      case Some(d) if d < new LocalDate("2013-04-01") => range2(car)
-      case Some(d) if d < new LocalDate("2015-04-01") => range3(car)
-      case Some(d) if d < new LocalDate("2018-04-01") => range4(car)
-      case Some(_) => range5(car)
+      case Some(dateOfPurchase) if dateOfPurchase < new LocalDate("2009-04-01") => range1(car)
+      case Some(dateOfPurchase) if dateOfPurchase < new LocalDate("2013-04-01") => range2(car)
+      case Some(dateOfPurchase) if dateOfPurchase < new LocalDate("2015-04-01") => range3(car)
+      case Some(dateOfPurchase) if dateOfPurchase < new LocalDate("2018-04-01") => range4(car)
+      case Some(dateOfPurchase) if dateOfPurchase < new LocalDate("2021-04-01") => range5(car)
+      case Some(dateOfPurchase) if dateOfPurchase < new LocalDate("2025-04-01") => range6(car)
+      case Some(dateOfPurchase) if dateOfPurchase >= new LocalDate("2025-04-01") => mainRate
```

Comment:
We still need main rate after 2025. You need to make a range7 which is literally just need range6 without the FYA line

Reply:
Is that a comment around following the existing pattern? Or is there a case I'm missing?